packages/quantic/force-app/examples/main/lwc/exampleQuanticRefineToggle/templateWithFacets.html

```diff
@@ -106,7 +106,7 @@
           ></c-quantic-date-facet>
         </template>
       </c-quantic-facet-manager>
-      <div class="container slds-is-relative">
+      <div class="container slds-is-relative slds-var-p-around_small">
```

Comment:
the -var- classes are meant to change depending on the org's density setting right? Will it still all look how we want with different settings?